backend/main.cpp

```diff
@@ -179,7 +188,9 @@ void handle_request(const std::string &request, ip::tcp::socket &frontend_socket
             // Multiply matrices 1 and 2 and store the result in the result matrix
             matrices.result_matrix = matrices.input_matrix_1 * matrices.input_matrix_2;
         }
-
+        const auto now = std::chrono::high_resolution_clock::now();
+        const std::chrono::duration<double> diff = now - timer;
```

Comment:
nit: Make this in microsecs pls

long long diff_us = std::chrono::duration_cast<std::chrono::microseconds>(now - timer).count();
std::cout << "PROFILER - Request to result: " << diff_us << " microseconds." << std::endl;